### DISCUSSION TITLE
Assets not deleting

### DISCUSSION THREAD

**Discussion by cleggers21** [aaronwmorris/indi-allsky#1467] [Q&A] [answered]
Absolute noobie and trying to get my head around how everthing works. I'm sure I'll get there but a guide or tutorial for the basics would be good.
Anyway .... I'm stuck with trying to fix a timelapse failure last night. If I run a replacement it fails because one exists. I tried using the delete action but nothing gets deleted., still showing as [ ! T ]
I note in the info file that timelapse overwrite is 'false' but I can't see where I can change that.

**Comment by cleggers21** [comment]
Ah ha, found the overwrite option hiding in plain sight in 'Admin' so I've managed to generte a TL now.

**Reply by aaronwmorris** [reply, marked as answer]
Sometimes it is difficult to find the right spot to put the config items.  And indi-allsky has  a LOT of config options.
